spikeextractors/extractors/neuroscopesortingextractor/neuroscopesortingextractor.py

```diff
@@ -66,17 +94,18 @@ def write_sorting(sorting, save_path):
         if len(unit_ids) > 0:
             spiketrains = [sorting.get_unit_spike_train(u) for u in unit_ids]
             res = np.concatenate(spiketrains).ravel()
-            clu = np.concatenate([np.repeat(i+2,len(st)) for i,st in enumerate(spiketrains)]).ravel()
+            clu = np.concatenate([np.repeat(i+1,len(st)) for i,st in enumerate(spiketrains)]).ravel() # i here counts from 0
             res_sort = np.argsort(res)
             res = res[res_sort]
             clu = clu[res_sort]
         else:
             res = []
             clu = []
         # add fake 'unit 1'
-        clu = np.insert(clu, 0, 1)
-        res = np.insert(res, 0, 1)
-        clu = np.insert(clu, 0, len(unit_ids)+1)
+        # Cody: Why is this here? commenting it out for now
```

Comment:
Can you remove these comments?

Reply:
Yes we can delete this. This was for compatibility with an ancient tool that we were using.

Reply:
Sounds good